examples/basic/providers.tf

```diff
@@ -0,0 +1,63 @@
+# see https://registry.terraform.io/providers/hashicorp/awscc/latest/docs
+provider "awscc" {
+  # configuration for this provider is handled through environment variables
+  # see https://registry.terraform.io/providers/hashicorp/awscc/latest/docs#environment-variables
+
+  # TODO @ksatirli: add default_tags, once supported
+  #  default_tags {
+  #    tags = {
+  #      terraform_managed = "true"
+  #      terraform_module  = "aws-ia/terraform-module-awscc-dropshare"
+  #    }
+  #  }
+
+  user_agent = [{
+    product_name    = "terraform-module-awscc-dropshare"
+    product_version = "0.1.0"
+
+    # the ID can be retrieved using `gh`: `gh api repos/aws-ia/terraform-awscc-dropshare | jq .id`
+    # see https://github.com/aws-ia/standards-terraform/blob/main/content/faq/_index.en.md#what-is-the-user_agent
+    # note: the above link is currently only accessible to members of the `aws-ia` GitHub Organization
+    comment = "V1/AWS-D69B4015/459237564"
+  }]
+}
+
+# see https://registry.terraform.io/providers/hashicorp/awscc/latest/docs
+provider "aws" {
+  # configuration for this provider is handled through environment variables
+  # see https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication
+
+  # set this to the region you want to create the S3 Bucket in
+  region = "us-east-1"
+
+  # see https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags
+  # also see https://learn.hashicorp.com/tutorials/terraform/aws-default-tags
+  default_tags {
+    tags = {
```

Comment:
tags should be a variable

if you want to easily use the same tags for `awscc_` resources pass the new `var.tags` to the [label module](https://github.com/aws-ia/terraform-aws-label)

```terraform
module "tags" {
  source  = "aws-ia/label/aws"
  version = "0.0.4"

  tags = var.tags
}
```

then you can reference in awscc resources like (example concat'd with a static tag value):

```terraform
  tags = concat(
    [{"key" = "Name", "value" = var.name}],
      module.tags.tags
  )
```

Reply:
I believe `default_tags` isn't eligible for variables, as it is in the `providers` block.

On a per-resource basis, it would be possible to use this module.